### PR TITLE
Remove fn call

### LINKED ISSUE #510: [UI] Remove 'call' from the CLI Help
CLI (0.5.40) still mentions in its help `call` as one of the development commands (ex. `fn`).
Given this command is not supported anymore, it should simply be removed from the help.
`fn call` produces the appropriate "no longer supported" warning.

### PR DESCRIPTION
**Pull request by rikgibson**
One-liner to remove no-longer-supported `fn call`,

Fixes #510

### CODE REVIEW

File: commands/commands.go [hunk 1]
```diff
@@ -22,7 +22,7 @@ var Commands = Cmd{
 	"build":        BuildCommand(),
 	"build-server": BuildServerCommand(),
 	"bump":         common.BumpCommand(),
-	"call":         CallCommand(),
+//	"call":         CallCommand(),
```
Comment (rdallman) [inline commands/commands.go]: can we remove the command code as well here please?

File: .gitignore [hunk 2]
```diff
@@ -9,4 +9,5 @@ _vendor*/
 /cli
 data/
 .vscode/
-.idea/
\ No newline at end of file
+.idea/
+**/*.iml
```
Comment (denismakogon) [inline .gitignore]: worth to add to global git ignore, no?

File: commands/commands.go (hunk 1 above)
Comment (rikgibson) [reply commands/commands.go]: duh, more haste less speed.

File: .gitignore (hunk 2 above)
Comment (rikgibson) [reply .gitignore]: sure you can do that too, this does no harm though for anyone that hasn't done so.

**Pull request merged**